package.json

```diff
@@ -1,249 +1,253 @@
+		"lint": "eslint src --ext ts && npm run lint --prefix webview-ui",
+		"package": "npm run build:webview && npm run check-types && npm run lint && node esbuild.js --production",
+		"pretest": "npm run compile-tests && npm run compile && npm run lint",
+		"start:webview": "cd webview-ui && npm run start",
+		"test": "jest && npm run test:webview",
+		"test:webview": "cd webview-ui && npm run test",
+		"test:extension": "vscode-test",
+		"prepare": "husky",
+		"publish:marketplace": "vsce publish && ovsx publish",
+		"publish": "npm run build && changeset publish && npm install --package-lock-only",
+		"version-packages": "changeset version && npm install --package-lock-only",
+		"vscode:prepublish": "npm run package",
+		"vsix": "mkdir -p bin && npx vsce package --out bin",
+		"watch": "npm-run-all -p watch:*",
+		"watch:esbuild": "node esbuild.js --watch",
+		"watch:tsc": "tsc --noEmit --watch --project tsconfig.json",
+		"watch-tests": "tsc -p . -w --outDir out"
+	},
+	"devDependencies": {
+		"@changesets/cli": "^2.27.10",
+		"@changesets/types": "^6.0.0",
+		"@types/diff": "^5.2.1",
+		"@types/jest": "^29.5.14",
+		"@types/mocha": "^10.0.7",
+		"@types/node": "20.x",
+		"@typescript-eslint/eslint-plugin": "^7.14.1",
+		"@typescript-eslint/parser": "^7.11.0",
+		"@vscode/test-cli": "^0.0.9",
+		"@vscode/test-electron": "^2.4.0",
+		"dotenv": "^16.4.7",
+		"esbuild": "^0.24.0",
+		"eslint": "^8.57.0",
+		"husky": "^9.1.7",
+		"jest": "^29.7.0",
+		"jest-simple-dot-reporter": "^1.0.5",
+		"lint-staged": "^15.2.11",
+		"npm-run-all": "^4.1.5",
+		"ts-jest": "^29.2.5",
+		"typescript": "^5.4.5"
+	},
+	"dependencies": {
+		"@anthropic-ai/bedrock-sdk": "^0.10.2",
+		"@anthropic-ai/sdk": "^0.26.0",
+		"@anthropic-ai/vertex-sdk": "^0.4.1",
+		"@aws-sdk/client-bedrock-runtime": "^3.706.0",
+		"@google/generative-ai": "^0.18.0",
+		"@modelcontextprotocol/sdk": "^1.0.1",
+		"@types/clone-deep": "^4.0.4",
+		"@types/pdf-parse": "^1.1.4",
+		"@types/turndown": "^5.0.5",
+		"@types/vscode": "^1.95.0",
+		"@vscode/codicons": "^0.0.36",
+		"axios": "^1.7.4",
+		"cheerio": "^1.0.0",
+		"chokidar": "^4.0.1",
+		"clone-deep": "^4.0.1",
+		"default-shell": "^2.2.0",
+		"delay": "^6.0.0",
+		"diff": "^5.2.0",
+		"fast-deep-equal": "^3.1.3",
+		"globby": "^14.0.2",
+		"isbinaryfile": "^5.0.2",
+		"mammoth": "^1.8.0",
+		"monaco-vscode-textmate-theme-converter": "^0.1.7",
+		"openai": "^4.73.1",
+		"os-name": "^6.0.0",
+		"p-wait-for": "^5.0.2",
+		"pdf-parse": "^1.1.1",
+		"puppeteer-chromium-resolver": "^23.0.0",
+		"puppeteer-core": "^23.4.0",
+		"serialize-error": "^11.0.3",
+		"sound-play": "^1.1.0",
+		"strip-ansi": "^7.1.0",
+		"tree-sitter-wasms": "^0.1.11",
+		"turndown": "^7.2.0",
+		"web-tree-sitter": "^0.22.6",
+		"zod": "^3.23.8"
+	},
+	"lint-staged": {
+		"*.{js,jsx,ts,tsx,json,css,md}": [
+			"prettier --write"
+		],
+		"src/**/*.{ts,tsx}": [
+			"prettier --write",
+			"npx eslint -c .eslintrc.json"
```

Comment:
Should we make this a `eslint --fix`? And how in sync are these rules with prettier? Between tsc and prettier, which of these lint rules are critical?

Reply:
Yeah we can make it with the `--fix` parameter. I just copied the existing command we had already.

Reply:
In for a penny, in for a pound!